acl.txt

```diff
@@ -1,6 +0,0 @@
-[
```

Comment:
did you mean to delete this file?

Reply:
Yep. It's not used.

Reply:
it is used @vpetersson in the github actions workflows:

For example:
https://github.com/NebraLtd/hm-block-tracker/blob/cf3e75ac65f6ae3336c79c97128aafbb55b1db2e/.github/workflows/run-update-production.yml#L46